docs/src/pages/ui/components/authenticator/headless.angular.mdx

```diff
@@ -0,0 +1,35 @@
+The `Authenticator`'s UI is powered by a state machine. You can use the `StateMachineService`composable
+to get access to it and customizing behavior or your own UI:
+
+_app.component.ts_
+
+```ts
+import { StateMachineService } from '@aws-amplify/ui-angular';
+
+export class AppComponent {
+  constructor(private stateMachine: StateMachineService) {}
```

Comment:
This service will likely be changed with a more publicly facing API, e.g. `AuthenticatorService`.